README.md

```diff
@@ -156,7 +156,7 @@ To use the PRPN sentence encoder from [***Neural language modeling by jointly le
 
 We also include an experimental option to use a shared [Transformer](https://arxiv.org/abs/1706.03762) in place of the shared BiLSTM by setting ``sent_enc = transformer``. When using a Transformer, we use the [Noam learning rate scheduler](https://github.com/allenai/allennlp/blob/master/allennlp/training/learning_rate_schedulers.py#L84), as that seems important to training the Transformer thoroughly. 
 
-We also support using pretrained Transformer language models. To use the OpenAI transformer model, set `openai_transformer = 1`.
+We also support using pretrained Transformer language models. To use the OpenAI transformer model, set `openai_transformer = 1`, download the [model](https://github.com/openai/finetune-transformer-lm) folder that contains pre-trained models, and place it under `src/openai_transformer_lm/pytorch_huggingface/`.
```

Comment:
Why do we need people to manually download these? The huggingface code seems like it should be able to automatically download any needed weights (which is how we use BERT).

Reply:
Huggingface does not automatically download the OpenAI models. They ask to manually download the OpenAI models in their README.

Reply:
Got it, thanks! The readme makes it sound possible, but looking at the code, I don't see any URL.